custom_components/oura/sensor_activity.py

```diff
@@ -22,6 +22,7 @@
     'score',
     'target_calories',
     'total_calories',
+    'day',
```

Comment:
nit: The attributes are alphabetically ordered. Do you mind putting 'day' between 'active_calories' and 'high_activity_time'?